website/source/assets/stylesheets/_inner.scss

```diff
@@ -52,9 +54,9 @@
   img {
     display: block;
     margin: 25px auto;
-    max-width: 650px;
+    max-width: 95%;
     height: auto;
-    width: 90%;
+    width: auto;
```

Comment:
This allows images to be bigger; the 650px width is arbitrary and doesn't serve screenshots well.